src/Serilog.Sinks.OpenTelemetry/Serilog.Sinks.OpenTelemetry.csproj

```diff
@@ -2,7 +2,7 @@
 	<PropertyGroup>
 		<Description>This Serilog sink transforms Serilog events into OpenTelemetry
 			logs and sends them to an OTLP (gRPC or HTTP) endpoint.</Description>
-		<VersionPrefix>2.0.1</VersionPrefix>
+		<VersionPrefix>3.0.0</VersionPrefix>
 		<Authors>Serilog Contributors</Authors>
 		<TargetFrameworks>net462;netstandard2.0;net6.0</TargetFrameworks>
```

Comment:
does the 462 fall off or become 48? (asking coz cant recall what core does and I see the ItemGroup at L32 went away) (Assuming it comes tranistively etc, not worried, just wondering)